en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md

```diff
@@ -0,0 +1,262 @@
+                    <s3Bucket>{$ctx:s3Bucket}</s3Bucket>
+                    <s3Key>{$ctx:s3Key}</s3Key>
+                    <s3ObjectVersion>{$ctx:s3ObjectVersion}</s3ObjectVersion>
+                    <zipFile>{$ctx:zipFile}</zipFile>
+                    <targetArn>{$ctx:targetArn}</targetArn>
+                    <environmentVariables>{$ctx:environmentVariables}</environmentVariables>
+                    <handler>{$ctx:handler}</handler>
+                    <kmsKeyArn>{$ctx:kmsKeyArn}</kmsKeyArn>
+                    <layers>{$ctx:layers}</layers>
+                    <memorySize>{$ctx:memorySize}</memorySize>
+                    <publish>{$ctx:publish}</publish>
+                    <role>{$ctx:role}</role>
+                    <runtime>{$ctx:runtime}</runtime>
+                    <tags>{$ctx:tags}</tags>
+                    <timeout>{$ctx:timeout}</timeout>
+                    <mode>{$ctx:mode}</mode>
+                    <securityGroupIds>{$ctx:securityGroupIds}</securityGroupIds>
+                    <subnetIds>{$ctx:subnetIds}</subnetIds>
+                </amazonlambda.createFunction>
+                <respond/>
+            </inSequence>
+            <outSequence/>
+            <faultSequence/>
+        </resource>
+    </api> 
+    ```
+        
+2. Right-click on the Composite Application Project and click on **Export Project Artifacts and Run**. Select **Run on Micro Integrator**.
+3. Micro Integrator will be started and the composite application will be deployed. You can further refer to the application deployed through the CLI tool. Make sure you first export the PATH as below.
+
+    ```
+    $ export PATH=/path/to/mi/cli/directory/bin:$PATH\
+    ```
+{! /references/connectors/exporting-artifacts.md !}
+
+## Create Amazon Lambda Deployment Package (Lambda function) 
+In this scenario we created sample AWS Deployment Package (Lambda function) in Python.
+
+1. Our sample Deployment Package would look similar to the following (source view : addingNumbers.py).
+```
+import json
+   
+   print('Loading function')
+   
+   def addingNumbers(event, context):
+       #print("Received event: " + json.dumps(event, indent=2))
+       value1 = event['key1']
+       value2 = event['key2']
+       print("value1 = " + value1)
+       print("value2 = " + value2)
+       return float(value1) + float(value2) # Echo back the addition of two keys
+       #raise Exception('Something went wrong')
+```
+
+2. Create a ZIP archive.
+
+Please use command line terminal or shell to run following commands.Commands are shown in listings preceded by a prompt symbol ($) and the name of the current directory, when appropriate:
+```
+~/Documents$ zip addingNumbers.zip addingNumbers.py
+ adding: addingNumbers.py.py (deflated 17%)
+```
+## Upload Amazon Lambda Deployment Package (ZIP archive) in to the AWS S3 bucket
+1. Log in to the AWS Management Console.
+2. Navigate in to the created S3 bucket (Ex: eiconnectortest).
+3. Click Upload button.
```

Comment:
```suggestion
3. Click **Upload**.
```

Use bold for buttons, links, and other such items in the UI.

Reply:
Done.